tools/check/go.mod

```diff
@@ -20,3 +20,5 @@ require (
 	gopkg.in/yaml.v2 v2.2.2 // indirect
 	honnef.co/go/tools v0.0.0-20180920025451-e3ad64cb4ed3
 )
+
+replace golang.org/x/tools => golang.org/x/tools v0.0.0-20190829051458-42f498d34c4d
```

Comment:
why not directly update the version of `golang.org/x/tools`?

Reply:
updated, PTAL.